src/Orleans/Telemetry/Consumers/FileTelemetryConsumer.cs

```diff
@@ -43,5 +42,23 @@ public void TrackTrace(string message, Severity severity, IDictionary<string, st
         {
             TrackTrace(message, properties);
         }
+
+        public void Flush()
+        {
+            lock (_lockObj)
+            {
+                if (_logOutput == null) return;
+                _logOutput.Flush();
+            }
+        }
+
+        public void Close()
```

Comment:
THis is the code for the old Close method of the LogWriterToFile log consumer:

        public void Close()
        {
            if (logOutput == null) return; // was already closed.

            try
            {
                lock (lockObj)
                {
                    if (logOutput == null) // was already closed.
                    {
                        return;
                    }
                    logOutput.Flush();
                    logOutput.Close();
                }
            }
            catch (Exception exc)
            {
                string msg = string.Format("Ignoring error closing log file {0} - {1}", logFileName, TraceLogger.PrintException(exc));
                Console.WriteLine(msg);
            }
            this.logOutput = null;
            this.logFileName = null;
        }